server/cmd/api/Makefile

```diff
@@ -1,5 +1,8 @@
 init:
 	hz new -idl ./../../idl/api.proto -mod github.com/CyanAsterisk/FreeCar/server/cmd/api
 
-update:
-	hz update -I -idl ./../../idl/api.proto
\ No newline at end of file
+update-proto:
+	hz update -I -idl ../../idl/api.proto
+
+update-thrift:
+	hz update -idl ../../idl/api.proto
```

Comment:
It doesn't seem to be correct. `api.proto` is still under the command `update-thrift`. This project should not have `api. thrift`. You can consider deleting it.